README.md

```diff
@@ -22,12 +22,22 @@ sudo make install
 ```bash
 mkdir myproject && cd myproject
 
+For a GitHub repo:
```

Comment:
Note that these examples are inside a `bash` code block, and this line is not valid bash syntax.

Maybe it would be best to separate each example `bundle.json` into its own separate `bash` block?